sdk/nodejs/runtime/settings.ts

```diff
@@ -32,7 +32,7 @@ const grpcChannelOptions = { "grpc.max_receive_message_length": maxRPCMessageSiz
 /**
  * excessiveDebugOutput enables, well, pretty excessive debug output pertaining to resources and properties.
  */
-export let excessiveDebugOutput: boolean = false;
+export const excessiveDebugOutput: boolean = false;
```

Comment:
This might be substantive? 

```
`const` is a signal that the identifier won't be reassigned. `let` is a signal that the variable may be reassigned,
```

Will this prevent a user from reassigning this value?

Reply:
Good catch.

I think the original intention was to allow it to be assigned. Though, I suspect nobody is setting it these days (besides maybe a Pulumi engineer on a rare occasion).

With ES6 modules, you aren't allowed to reassign an exported value from outside the module (imports are a read-only live view of the value), so it might be worth considering changing it to `const` to discourage anyone setting it from outside the module going forward.

Changing it to `const` does mean it isn't useful unless you're modifying the source directly (either by modifying a copy of `@pulumi/pulumi` inside `node_modules` or using `yarn link` locally). If being able to set it is missed, we could consider adding a `setExcessiveDebugOutput` function, or perhaps better, switch to using an environment variable that could work across all SDKs.

Reply:
In that case let's accept this change for now.

Reply:
> If being able to set it is missed, we could consider adding a setExcessiveDebugOutput function, or perhaps better, switch to using an environment variable that could work across all SDKs.

Could we open an issue for it? I could also work on this issue as well as a follow up.